federation/pkg/dnsprovider/providers/coredns/rrsets.go

```diff
@@ -75,6 +75,8 @@ func (rrsets ResourceRecordSets) Get(name string) (dnsprovider.ResourceRecordSet
 			rrset.rrsType = rrstype.CNAME
 		case ip.To4() != nil:
 			rrset.rrsType = rrstype.A
+		default:
```

Comment:
Should I:

- Leave it as-is
- Add a comment indicating the default is for IPv6
- Add a case with test for To6(), and then have default case with comment indicating it should never occur?